dunwoody_disaster/CharacterFactory.py

```diff
@@ -125,12 +127,10 @@ def createCharacter(name: str, classType: str) -> Character:
         character.classType = classType
         character.maxHealth = data["health"]
         character.maxMagic = data["magic"]
-        character.maxMechanical = data["mechanical"]
+        character.maxStamina = data["mechanical"]
         character.curHealth = data["health"]
         character.curMagic = data["magic"]
-        character.curMechanical = data["mechanical"]
-        character.defense = data["defense"]
-        character.magicDefense = data["magicDefense"]
+        character.curStamina = data["mechanical"]
```

Comment:
We should phase out `data["mechanical"]`. The `CharacterFactory` is going to need some tweaks.

Reply:
I see  that you changed it to use `data['stamina']`, but you also need to change it on `CharacterFactory.class_types`